didpass-demo/issuer-demo/src/components/UserDropdown/index.tsx

```diff
@@ -0,0 +1,190 @@
+import ClickAwayListener from "@mui/material/ClickAwayListener";
+import Grow from "@mui/material/Grow";
+import MenuItem from "@mui/material/MenuItem";
+import MenuList from "@mui/material/MenuList";
+import Paper from "@mui/material/Paper";
+import Popper from "@mui/material/Popper";
+import Stack from "@mui/material/Stack";
+import Cookies from "js-cookie";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import * as React from "react";
+import { getToken, removeUserCookie } from "@/utils/cookie";
+import { OutlinedButton } from "../Button/OutlinedButton";
+import Toast, { ToastConfig } from "../Toast";
+import styles from "./UserDropdown.module.css";
+
+const UserDropdown = () => {
+  const [open, setOpen] = React.useState(false);
+
+  const anchorRef = React.useRef<HTMLButtonElement>(null);
+  const userCookie = Cookies.get("_user");
+  const [wildcardDid, setWildcardDid] = React.useState("");
+  const [last4chars, setLast4chars] = React.useState("");
+  const [toastConfig, setToastConfig] = React.useState<ToastConfig>({
+    message: "",
+    severity: "success",
+    open: false,
+  });
+  const router = useRouter();
+
+  React.useEffect(() => {
+    if (userCookie) {
+      let originalString = JSON.parse(userCookie).did;
+      let last4chars = originalString.slice(-4);
+      let prefixLength = originalString.lastIndexOf(":") + 1;
+      let wildcardDid =
+        originalString.substring(0, prefixLength) + "..." + last4chars;
+      setLast4chars(last4chars);
+      setWildcardDid(wildcardDid);
+    }
+  }, []);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = (event: Event | React.SyntheticEvent) => {
+    if (
+      anchorRef.current &&
+      anchorRef.current.contains(event.target as HTMLElement)
+    ) {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  const disconnect = async () => {
+    const token = getToken();
+    await fetch(`/api/ssi/issuer/disconnect`, {
```

Comment:
@handrianalandi ini kayanya endpointnya belom ada deh

Reply:
oh iyaa kak my bad, working on it